srcpkgs/m4ri/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'm4ri'
+pkgname=m4ri
+version=20200125
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr --with-openmp"
+depends="libpng"
```

Comment:
No need for this (automatic by soname), but instead you need `makedepends=libpng-devel`.